### PR TITLE
Removing acronyms from the cover letter and making a few modifications

### PR DESCRIPTION
**Pull request by pedrohbraga**
Hi, 

I made a few modifications to the cover letter to remove the acronyms (which did not appear essential and could bother the editorial) and suggested some changes to the content.

### CODE REVIEW

File: editorial-communication/cover-letter-submission.Rmd [hunk 1]
```diff
@@ -11,24 +11,24 @@ output:
 ```{r include=FALSE}
 knitr::opts_chunk$set(fig.retina = 2)
 ```
+
 Dear Editor,
 
 We would like to submit our manuscript entitled “Not just for programmers: A friendly guide on the benefits of GitHub for accelerating collaborative research in Ecology and Evolution” for consideration to be published in *Nature Ecology & Evolution*.
```
Comment (DrMattG) [inline editorial-communication/cover-letter-submission.Rmd:17]: Note change in title "Not just for programmers: How GitHub can accelerate collaborative and reproducible research in ecology and evolution"

File: editorial-communication/cover-letter-submission.Rmd [hunk 2]
```diff
@@ -11,24 +11,24 @@ output:
...
-Our manuscript is important for the *Nature Ecology & Evolution* readership because it will give EEB researchers key resources and workflows to guide them through migrating workflows to GitHub and supporting more open and collaborative science.
+Our manuscript is the most accessible and practical guide to using GitHub in ecology and evolution research to date.
+We build upon our own experiences (as an international collaboration from authors across many ecology and evolution subdisciplines) and on existing resources mainly targeted at other disciplines, to highlight how researchers can use the readily available GitHub suite to make their research practices reproducible, collaborative, and transparent.
+Our manuscript is important for the *Nature Ecology & Evolution* readership because researchers in ecology and evolutionary biology can greatly benefit from our guide to migrate and incorporate GitHub workflows in their research practices ultimately towards an open and collaborative science.
 
 The enclosed manuscript is available as a preprint on the NAME OF SERVER AND CITATION, but is not published or in review elsewhere.
```
Comment (DrMattG) [inline editorial-communication/cover-letter-submission.Rmd:33]: A general question (@robcrystalornelas) should we preprint earlier than we submit or simultaniously - I'm thinking if we do it a few weeks beforehand we can push the preprint on twitter and elsewhere and then in the cover letter can say something along the lines of " our preprint has been downloaded XXX times in two weeks illustrating how important it is for our community"... just a thought.

**Review by DrMattG — APPROVED** [review]
Looks good - just note the change in title

File: editorial-communication/cover-letter-submission.Rmd (hunk 1 above)
Comment (robcrystalornelas) [reply editorial-communication/cover-letter-submission.Rmd]: Thanks for catching that. I updated this PR with our new title

**Pull request merged**